crates/ruff_workspace/src/configuration.rs

```diff
@@ -1502,7 +1502,7 @@ mod tests {
         Rule::ReimplementedStarmap,
         Rule::SliceCopy,
         Rule::TooManyPublicMethods,
-        Rule::TooManyPublicMethods,
+        Rule::TooManyAncestors,
```

Comment:
I'm assuming this wasn't meant to be a duplicate, but for visibility I'm adding this comment.